odoo/http.py

```diff
@@ -181,7 +181,6 @@
                     profiler, submap, unique, ustr,)
 from .tools.func import filter_kwargs, lazy_property
 from .tools.mimetypes import guess_mimetype
-from .tools.misc import pickle
```

Comment:
Should we remove (or deprecated at least) our pickle version from the misc.py ? AFAIK, you remove the last usage.